packages/faker/index.ts

```diff
@@ -33,6 +33,8 @@ const suite2 = newId(3)
 const suite3 = newId(4)
 const suite4 = newId(5)
 const suite5 = newId(6)
+const suite6 = newId(7)
+const suite7 = newId(8)
```

Comment:
I think we can have predicable results with faker using the seed feature: https://github.com/marak/Faker.js/#setting-a-randomness-seed